### ISSUE TITLE
Support kroki diagram options

### ISSUE THREAD

**Issue by p-rogalski** [pomdtr/vscode-markdown-kroki#4]
Kroki allows to pass options to diagrams:
How-to use: https://docs.kroki.io/kroki/setup/usage/#_options
Option Reference: https://docs.kroki.io/kroki/setup/diagram-options/

Use Case:
Mermaid has a dark mode, which could be automatically set when your extension senses VSCode dark mode. Something similar is done by https://github.com/mjbvz/vscode-markdown-mermaid

**Comment by pomdtr** [comment]
Great idea ! I'll take a look

**Comment by pomdtr** [comment]
I'm not super interested in introducing package specific features (ex: dark mode for mermaid files)

However, it could be interesting to allow setting these options using a yaml meta block:

````md
---
mermaid:
    theme: dark
---

```mermaid
graph TD;
    A-->B;
    A-->C;
    B-->D;
    C-->D;
```
```

mermaid
````

**Comment by p-rogalski** [comment]
> I'm not super interested in introducing package specific features (ex: dark mode for mermaid diagrams only).

I understand that. What about setting options in VSCode settings in a generic way and sending them each time a kroki request is made? VSCode settings could look something like:

```json
"markdown-kroki.options": {
  "key":"value"
}
```

> However, it could be interesting to allow setting these options using a yaml meta block

This could also work

**Comment by pomdtr** [comment]
Good point. Adding support for it both at a global, workspace and document level would be nice.
